src/main/java/org/dependencytrack/util/AnalysisCommentUtil.java

```diff
@@ -22,55 +22,63 @@
 import org.dependencytrack.model.AnalysisJustification;
 import org.dependencytrack.model.AnalysisResponse;
 import org.dependencytrack.model.AnalysisState;
-import org.dependencytrack.persistence.QueryManager;
+import org.dependencytrack.persistence.jdbi.AnalysisDao;
 import org.dependencytrack.util.AnalysisCommentFormatter.AnalysisCommentField;
 
+import static org.dependencytrack.persistence.jdbi.JdbiFactory.withJdbiHandle;
 import static org.dependencytrack.util.AnalysisCommentFormatter.formatComment;
 
 public final class AnalysisCommentUtil {
 
     private AnalysisCommentUtil() { }
 
-    public static boolean makeStateComment(final QueryManager qm, final Analysis analysis, final AnalysisState analysisState, final String commenter) {
```

Comment:
If you'd like to have these methods use JDBI, it's best to accept a JDBI `Handle` as parameter. Otherwise, multiple invocations of these methods could never share a transaction.

The following should be possible:

```java
useJdbiTransaction(handle -> {
    AnalysisCommentUtil.makeStateComment(handle, ..., ...);
    AnalysisCommentUtil.makeJustificationComment(handle, ..., ...);
    // ...
});
```

In fact, you might want to consider moving all these methods to `AnalysisDao` instead.